src/routes/oauth2/middleware.test.js

```diff
@@ -126,15 +129,29 @@ describe("oauth middleware", () => {
           get: sinon.stub(),
         };
         req.app.get.withArgs("API.PATHS.SESSION").returns("/api/authorize");
+        req.app.get.withArgs("API.BASE_URL").returns("http://localhost:3000");
+
+        req.headers = {
+          "txma-audit-encoded": "dummy-txma-header",
+          "x-forwarded-for": "198.51.100.10:46532",
+        };
       });
 
       it("should call axios with the correct parameters", async function () {
         await middleware.initSessionWithJWT(req, res, next);
-
-        expect(req.axios.post).to.have.been.calledWith("/api/authorize", {
-          request: exampleJwt,
-          client_id: req.session.authParams.client_id,
-        });
+        expect(req.axios.post).to.have.been.calledWith(
+          "/api/authorize",
+          {
+            request: exampleJwt,
+            client_id: req.session.authParams.client_id,
+          },
+          {
+            headers: createPersonalDataHeaders(
```

Comment:
Do we want to maybe mock `createPersonalDataHeaders`  rather than using it directly in the tests?

Reply:
I think it's better to use the original plugin. We want to be sure that is working as expected.
@nickhealGDS, what is your opinion?

Reply:
But we are using the plugin in the test so if the plugin is broken and it will result in a broken expectation and the tests will still pass?

Reply:
In same time, if we hardcode the expecting personal data header in the unit tests instead to use the function and tomorrow the function is updated and returns more headers, the test will failed.
If the function doesn't return the good headers, the test will still pass.
Anyway, @nickhealGDS prefers to not use the function into the unit test. So I updated it.
Thanks @Caitlin-cooling for your feedbacks!